site/code-snippets/web5/build/decentralized-web-nodes/use-identity-agents.js

```diff
@@ -0,0 +1,48 @@
+import { IdentityAgent } from '@web5/identity-agent';
+import { getTechPreviewDwnEndpoints } from '@web5/api';
+import { DidIonMethod } from '@web5/dids';
+
+
+export async function createIdentityAgent() {
+    const agent = await IdentityAgent.create();
+    return agent;
+}
+
+export async function authenticateIdentityAgent(agent) {
+    await agent.start({ passphrase: 'default-passphrase' });
+    return agent.agentDid;
+}
+
+export async function getDwnEndpoints() {
+// selects DWN endpoints that are provided by default during the Web5 tech preview period
+const serviceEndpointNodes = await getTechPreviewDwnEndpoints();
+// generates key pairs used for authorization and encryption when interfacing with DWNs
```

Comment:
```suggestion

// generates key pairs used for authorization and encryption when interfacing with DWNs
```

added a blank line for readability